src/demos/components/properties.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import PropertyJson from "./components_properties.json";
+import { Grid, Inner, Cell } from "../../../src/libs";
+
+const Head = ["Name", "Default", "Type"];
+
+function Property(Component) {
```

Comment:
What's that? That's not exactly how we write React components. `Component` is not a component but `props` because what you are describing here is a functional component.

The way we write such components is as follow:

```js
const Property = (props) => {
  ...
  return (
    <p /> 
  );
};

export default Property;
```

Also the file name should be the same as the component. Your component is called `Property`, so the filename should be `property.jsx`.